src/Graphics/Canvas.purs

```diff
@@ -121,10 +118,7 @@ foreign import data CanvasElement :: *
 foreign import data Context2D :: *
 
 -- | An image data object, used to store raster data outside the canvas.
-foreign import data ImageData :: *
-
--- | An array of pixel data.
-foreign import data CanvasPixelArray :: *
+type ImageData = { width :: Number, height :: Number, data :: Uint8ClampedArray }
```

Comment:
Is the underlying representation of Int still just a Number? I'm not sure whether I can just declare the type here and it works or if I need to actually coerce the Number to an Int.